app/assets/stylesheets/_main.scss

```diff
@@ -0,0 +1,43 @@
+.main {
+  margin: 46px auto 0;
+  width: 325px;
```

Comment:
I have updated this as max-width: 325px for the time being. I had a chat with Amanda who suggested the best way to gauge how wide the container should be is to figure out what the widest container for all views will be and style consistently. Since we do not yet have mocks for the other views, she has offered to do some for us :)